### DISCUSSION TITLE
Allow more directory organizations for partials

### DISCUSSION THREAD

**Discussion by acalvino4** [ExpressionEngine/ExpressionEngine#3109] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
The _partials directory can get quite cluttered. I would really appreciate the ability to have more organization rather than all partials just being dumped into a single folder.

**Describe the solution you'd like**
Possible solutions:
* allow subdirectories of the partials folder. referencing nested partials would just look like `{folder/partial}`
* allow specifying any path in the templates directory. referencing these partials might look like `{partial=folder/partial}` (similar to embeds)
* allow a configuration somewhere from which I can specify custom partial directories in addition to or in replacement of `_partials`

**Describe alternatives you've considered**
The alternative is sticking with the status quo - a cluttered, non-organized partials directory.

**Teachability, Documentation, Adoption, Migration Strategy**
Depending on what solution above is selected, we would just document it on the partials page of the documentation. Pretty Simple

- [x] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by robsonsobral** [comment]
This could be a good part of https://github.com/ExpressionEngine/ExpressionEngine/issues/1296

**Comment by shbchk** [comment]
Can't wait for this feature to be implemented. I have so much partials...

**Comment by swierczek** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4648 PR created 🤞
